nrf_security/src/psa_crypto_driver_wrappers.c

```diff
@@ -2750,4 +3034,121 @@ psa_status_t psa_driver_wrapper_asymmetric_decrypt(const psa_key_attributes_t *a
     }
 }
 
+#if defined(CONFIG_PSA_CORE_OBERON)
+
+psa_status_t psa_driver_wrapper_init_random(
+    psa_driver_random_context_t* context)
+{
+#if defined(PSA_CRYPTO_DRIVER_GENERATE_RANDOM_CC3XX_PLATFORM)
+    /* Using internal context. */
+    (void)context;
+
+    return PSA_SUCCESS;
+#endif
+
+#if defined(PSA_CRYPTO_DRIVER_ALG_CTR_DRBG_OBERON)
```

Comment:
Please continue the pattern of #if #elif #elif #endif similar to other functions for random generation, for consistency